src/parser/Parser.ts

```diff
@@ -401,7 +412,7 @@ export class Parser {
                 );
             }
             let endFor = advance();
-            while(match(Lexeme.Newline));
+            while (match(Lexeme.Newline));
```

Comment:
One of these days I'll set up `tsfmt` or something like that.

Reply:
Dang, I missed that one. I'm a very aggressive auto-formatter, so the last step in my commits is to undo all the auto-formats. lol Do you want me to undo this?

Reply:
Eh, doesn't bother me much either way right now.  It's a sensible change so I don't mind leaving it!  It was more a note to myself :man_shrugging: